stories/Pagination/Pagination.story.jsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import styled from 'styled-components';
+import {
+  TabbedView,
+  Tab,
+  AutoPropsApi,
+  Heading,
+  StoryContainer,
+  Title,
+  SimpleHighlight,
+  Table,
+} from '@catho/quantum-storybook-ui';
+
+import { Pagination, Row, Col } from '../../components';
+import Mobile from '../../components/Pagination/sub-components/Mobile';
+import ControlledPagination from './examples/ControlledPagination';
+import ControlledPaginationAria from './examples/ControlledPaginationAria';
+
+const MobileWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+`;
+
+storiesOf('Pagination', module).add('Pagination', () => (
+  <>
+    <Heading name="Pagination">
+      Add pagination buttons to help split up your long content into shorter,
+      easier to understand blocks.
+    </Heading>
+    <TabbedView>
+      <Tab title="Usage">
+        <StoryContainer>
+          <Title as="h2">Importing Pagination</Title>
+          <SimpleHighlight>{`import { Pagination } from '@catho/quantum';`}</SimpleHighlight>
+
+          <p>
+            An adaptive Pagination component, if viewport width is below small
```

Comment:
```suggestion
             The Pagination component will render a mobile-friendly component if the viewport width is below small breakpoint value else it will render a dotted component.
```

:nail_care:

Reply:
Done.
Thanks!